net/test/node_bulk.c

```diff
@@ -1277,17 +1283,23 @@ static void node_bulk_worker(struct node_bulk_ctx *ctx)
 			m0_net_buffer_event_notify(ctx->nbc_net.ntc_tm,
 						   &tm_chan);
 		}
+	#ifndef ENABLE_LIBFAB
 		ctx->nbc_callback_executed = false;
+	#endif
 		/* execute network buffer callbacks in this thread context */
 		m0_net_buffer_event_deliver_all(ctx->nbc_net.ntc_tm);
 		M0_ASSERT(ergo(pending, ctx->nbc_callback_executed));
+	#ifndef ENABLE_LIBFAB
 		/* state transitions from final states */
 		node_bulk_state_transition_auto_all(ctx);
+	#endif
 		/* update copy of statistics */
 		m0_net_test_nh_sd_copy_locked(&ctx->nbc_nh);
+	#ifndef ENABLE_LIBFAB
 		/* wait for STOP command or buffer event */
 		if (!ctx->nbc_callback_executed)
 			m0_chan_wait(&ctx->nbc_stop_clink);
+	#endif
```

Comment:
Why is the checking of "ENABLE_LIBFAB" needed?

Reply:
There are no pending buffer event callbacks in libfabric.
Hence had to disable this as the ctx->nbc_callback_executed was falsely being reset in the test.